partials/expenses_input.html

```diff
@@ -14,7 +16,7 @@
                     class="steps"
                     min="0"
                     max="99999"
-                    ng-model="expenses_ctrl.expenses" />
+                    ng-model="expenses" />

```

Comment:
Yes exactly, as long as its $scope.someObjext.someValue it should be fine
On Aug 29, 2015 12:44 PM, "Nathan Kane" <notifications@github.com> wrote:

> In partials/expenses_input.html
> <https://github.com/bobo333/financial_freedom/pull/80#discussion_r38265244>
> :
>
> > @@ -14,7 +16,7 @@
> >                      class="steps"
> >                      min="0"
> >                      max="99999"
> > -                    ng-model="expenses_ctrl.expenses" />
> > +                    ng-model="expenses" />
>
> Do you mean just make the value a property of an object? Like, in the
> controller, it'd be defined as $scope.userData.expenses ? The reason I had
> as part of an object before ( expneses_ctrl.expenses) was to avoid
> overusing $scope. But, I had to change it back because I needed to add a
> $watch to the value so that even when the user advances in the flow without
> submitting, the value is stored in UserDataCache. So, I could just use
> $scope.someObject.value and solve both issues.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bobo333/financial_freedom/pull/80/files#r38265244>.
>